README.md

```diff
@@ -29,3 +29,41 @@ Setup Database
 pnpm supabase migration up
 pnpm supabase db reset --debug
 ```
+
+### Postgres Functions and Triggers for Speakers
+
+You can create the following functions/triggers on SQL Editor.
+
+`add_user_to_speaker`: Function for trigger_add_user_to_speaker
+
+```
+create or replace function add_user_to_speaker()
+returns trigger
+language plpgsql
+security definer set search_path = ‘’
+as $$
+begin
+  insert into public.speakers(uuid, name, email)
+  values (new.id, new.email, new.email);
```

Comment:
Does `new.id` mean auth schema users.uuid?

![image](https://github.com/my-joshu/my-joshu/assets/7115171/46aeb93a-78a4-4270-aa98-86c628d52827)

Reply:
Yes it does. I intend to replace it with `uuid_generate_v4()` since it has no particular meaning. I overlooked it, thank you

This is a related topic though, `speakers.email` does not have a unique constraint right now. I think it would be nice to have for validation. Another PR will do it

Reply:
> This is a related topic though, speakers.email does not have a unique constraint right now. I think it would be nice to have for validation. Another PR will do it

Created a new issue https://github.com/my-joshu/my-joshu/issues/6